plugins/postcss-gradients-interpolation-method/test/basic.expect.css

```diff
@@ -129,7 +129,7 @@
 }
 
 .test-visual-effect-9-1-c {
-	background-image: linear-gradient(rgb(245, 137, 137) 0%, rgb(245, 133, 166), rgb(244, 129, 198), rgb(243, 124, 231), rgb(218, 120, 242), rgb(179, 116, 241), rgb(137, 111, 240), rgb(107, 120, 240), rgb(103, 157, 239), rgb(99, 196, 238), rgb(94, 237, 237) 100%);
+	background-image: linear-gradient(rgb(245, 137, 137) 0%, rgb(245, 166, 133), rgb(244, 198, 129), rgb(243, 231, 124), rgb(218, 242, 120), rgb(179, 241, 116), rgb(137, 240, 111), rgb(107, 240, 120), rgb(103, 239, 157), rgb(99, 238, 196), rgb(94, 237, 237) 100%);
```

Comment:
`longer` goes around the hue wheel in the different direction because rounding now favors this.

The new output matches Chrome Canary.
(To the best of my knowledge Chrome currently has the best implementation of everything color.)